src/web/mjs/connectors/templates/CoreView.mjs

```diff
@@ -34,7 +39,9 @@ export default class CoreView extends Connector {
         let data = await this.fetchDOM(request, this.queryManga);
         return data.map(element => {
             return {
-                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
+                id: this.queryMangaURI ?
```

Comment:
May inline ternary operator into parameter query
`id: this.getRootRelativeOrAbsoluteLink(this.queryMangaURI ? element.querySelector(this.queryMagaURI) : element, request.url)`

Reply:
`queryMangaURI` is a CSS selector.